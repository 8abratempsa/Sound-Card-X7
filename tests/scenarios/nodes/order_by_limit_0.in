octosql "SELECT * FROM range(start=>1, end=>10) r ORDER BY i DESC LIMIT 4"
