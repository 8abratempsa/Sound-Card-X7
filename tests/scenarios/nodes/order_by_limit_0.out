+-----+
| r.i |
+-----+
|   9 |
|   8 |
|   7 |
|   6 |
+-----+
